tests/BookFixture.php

```diff
@@ -17,12 +17,6 @@ class BookFixture
         "templates": {
             "foo": "/foo.php"
         },
-        "extensions": {
```

Comment:
Probably we can bring this back. Currently I have added these extensions by default.